src/raylib.h

```diff
@@ -261,7 +261,8 @@
 //----------------------------------------------------------------------------------
 #ifndef __cplusplus
 // Boolean type
-typedef enum { false, true } bool;
+#include <stdbool.h>
```

Comment:
Hi @kd7tck! Why do you need the stdbool.h? It works good with the typedef...

I did it that way for several reasons:

1) MSVC doesn't support this library until VS 2013 (http://stackoverflow.com/questions/8548521/trying-to-use-include-stdbool-h-in-vs-2010)
2) It's a new library from C99, C89 and C90 didn't support it. Despite raylib uses a minimal set of features of C99, I try to keep the full base-code as portable as possible.
3) It's easy to implement without the need of an external library (http://stackoverflow.com/questions/1921539/using-boolean-values-in-c). In raylib I always try to keep dependencies to the minimum.
4) I try to keep raylib.h clean of includes...